en_ID/pages/faq.md

```diff
@@ -11,8 +11,68 @@
 ### What are the eligibility requirements for participation?
 * Generally, there are no strict requirements. Anyone who early in their software engineering career can join this program as long as can commit a significant amount of time to this internship as we mention in the [Duration and time commitment](internship.md#Duration_and_time_commitment). This internship program is intensive, so a time commitment is very important as this program is running virtually.
 
+### How to Make a PR (Pull Request) ?
+
+The very first thing to do before making a PR is creating a new branch for your PR.
+
+The important point for creating branch is CREATE BRANCH FROM YOUR MASTER BRANCH.
+
+* Open the Terminal
+* Go to your project directory
+* Check current status.
+
+```
+git status
+```
+
+![][branch-image]
+
+* Be sure to be on the master Branch of your project.
+* Then create a new branch from master branch.
+
+```
+git checkout -b [your branch dir/your branch name]
+```
+* This command will create a new branch and bring you to this branch.
+* Then do your changes on project.
+* Check status.
+
+```
+git status
+``` 
+* You should see your changes.
+* Add this changes.
+
+```
+git add .
+```
+* Commit your changes on your local repo. Always write comment on your commits.
+
+```
+git commit -m "[your message]"
+```
+
+* Commit your changes to remote repo (at Github.com).
+
+```
+git push origin [your branch dir/your branch name]
+
+``` 
+* As you are using SSH (made on [Step 2](https://open-source.kulkul.tech/en_ID/#!pages/internship.md#Step_2_-_Learn_how_to_use_SSH), this step may ask you your pass (in case you defined))
+* Then you can check github page and see your changes pushed on the server.
+* Github automatically ask you if you want to create PR with your changes.
+* If you cant see automatic PR from github;
+* Click branch drop down list
+![][branchlist-image]
```

Comment:
one more challenge, the image displayed too big on my side, it doesn't fit with its container.

hint: you can use plain HTML to display images.

![Screen Shot 2021-01-19 at 10 04 23](https://user-images.githubusercontent.com/11813607/104983248-fdf38800-5a3e-11eb-9d2f-9f42669b3e68.png)

Reply:
Can you check it again ?